install/distro-independent.cfg

```diff
@@ -10,7 +10,7 @@ command = command -v pip2 >/dev/null || { wget https://bootstrap.pypa.io/get-pip
 
 [Httprint]
 directory = %(RootDir)s/tools/restricted/httprint
-command = wget "http://www.net-square.com/zip folders/httprint_linux_301.zip"; unzip *.zip; rm -f *.zip
+command = wget http://www.net-square.com/_assets/httprint_linux_301.zip; unzip *.zip; rm -f *.zip
```

Comment:
Shouldn't there be a semicolon at the end as there is another command at the end?